coresdk/src/test/test_logging.cpp

```diff
@@ -0,0 +1,15 @@
+#include "logging.h"
+
+using namespace splashkit_lib;
+
+void run_logging_test ()
+{
+    log_level _log_level = INFO;
+log_mode _log_mode = CONSOLE_AND_FILE;
```

Comment:
Watch indentation 😄

Reply:
@Macite, Thanks so much for the input, helps a lot!
1. I'll try and remember the naming convention for commit messages.
2. Removed break after return.
3. What do you mean by `init_sk`? A recursive `git grep` turned up nothing, unless I did it wrong...
4. RE switch-case: I'll change that as well.
5. RE checking log mode: I think I know what you mean, we'll see what we can do.
6. RE `NONE` option: yeah that's probably a good idea.
7. RE default arguments: Do you mean no default args within Splashkit per se? So consequently, always overload every procedure / function?
8. RE proposal of `void <level>_log (string message)`: It feels cleaner to me to have one function and then perform the appropriate operations based on the param inputs.
9. RE Location of log file: can do that.  Just thought current implementation might have been a set convention.
10. Will fix indentation.

Reply:
1. 👍 
2. 👍 
3. Sorry was working from memory. When splash kit is used for the first time it runs an "init" process to initialise things. Have a look at Core Driver - `internal_sk_init` and there is already a `sk_init_logging`. So you could locate and update that.
4. 👍 
5. 👍 
6. 👍 
7. Yes, splash kit functions cannot have default arguments as we cannot support this in the translator at the moment. So you will need to write out the different options as explicit overloads of the function, but they will all just call the most explicit version - so still just one implementation.
8. Sounds fine - happy to just keep the one function name.
9. I'll have to look at where else this is, but in this case keeping the path simple is probably best.
10. 👍 

Looking good!